src/locales/lang/fr.json

```diff
@@ -11,9 +11,9 @@
   "plugins.links.standardHint": "Cliquez pour consulter le site",
   "plugins.search.placeholder": "Taper pour rechercher",
   "plugins.unsplash.photoLink": "Photo",
-  "plugins.weather.high": "High",
-  "plugins.weather.low": "Low",
-  "settings": "Settings",
-  "settings.translationCredits": "Traduction française par Orfaon",
-  "widgets": "Widgets"
+  "plugins.weather.high": "Max",
+  "plugins.weather.low": "Min",
+  "settings": "Préférences",
```

Comment:
When I change my phone's settings to (European) French, "Settings" is translated to "Réglages". I'd guess this is the terrminology most Francophone people are used to.

Reply:
> 
> 
> When I change my phone's settings to (European) French, "Settings" is translated to "Réglages". I'd guess this is the terrminology most Francophone people are used to.

Yes this is used too :-)